src/utils/embedding_store.py

```
@@ -0,0 +1,425 @@
+
+                        latest_summary = await get_summary(
+                            db, self.workspace_name, session_name, SummaryType.SHORT
+                        )
+                        if latest_summary:
+                            summary_id = (
+                                str(latest_summary.get("message_id"))
+                                if latest_summary.get("message_id")
+                                else None
+                            )
+                            summary_content = latest_summary.get("content")
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)